helm/standalone-helm-chart-pipeline-template.yaml

```diff
@@ -41,7 +41,9 @@ steps:
         for i in ${childCharts[@]}; 
         do 
         echo "updating $i"
-        helm dependency update  $i; 
+        helm dependency update  $i;
+        status=$?
+        [ $status -eq 0 ] && echo "dep up on $i command was successful" || echo "dep up on $i failed"
```

Comment:
Will this still fail the build?

Reply:
not yet, i wanted to test this out before i have it fail the build

Reply:
But no change in current behavior

Reply:
sorry, our suspicion is that it isn't failing the build now.

Reply:
We had a release go out without many of the claimant portal apps because the ACR was giving us 503s